.github/workflows/main.yml

```
@@ -63,7 +63,7 @@ jobs:
       - name: Check for missing migrations
         working-directory: afs
         run: |
-          PYTHONPATH="../arches" python manage.py makemigrations --check --settings=arches_for_science.test_settings
+          PYTHONPATH="../arches" python manage.py makemigrations --check --settings=tests.test_settings
```

Let's remove the path manipulation workaround -- it was only here because the previous test settings file was much smaller and just imported test settings from arches.tests.test_settings.

Also, let's just use the normal settings file now.

```suggestion
          python manage.py makemigrations --check
```